### ISSUE TITLE
[Bug] favicon.ico will not apply from local store, but does apply the same ico from web source

### ISSUE THREAD

**Issue by TopCheddar27** [benphelps/homepage#595]
### Description

While customizing the favicon in settings, locally stored .ico files in the /app/public/images folder will not apply. However, when referencing the same .ico file from the web source, it applies just fine. First, I thought it could be permissions or storage location. However, background images sourced from the same /app/public/images folder with the exact same permissions apply just fine. I have also browsed locally in the container to confirm that the file is present within the folder specified to check for mounting issues within docker.

I can apply this same exact .ico file fine in many other applications I have, but I thought I would try another file to see if it's somehow "non standard". I first used the url from the documentation [here](https://gethomepage.dev/en/configs/settings/#favicon). Using the URL works. When downloading that same source .ico locally, with correct permissions, it will not apply.

For reference. the ico file I am trying to use is located here: https://iconarchive.com/download/i103430/paomedia/small-n-flat/house.ico

Here is a screenshot from when it works when a url is provided in settings.yaml, essentially proving that the file can work

![image](https://user-images.githubusercontent.com/73973868/204941918-0c2f6105-a7cf-44c4-93e0-69751dbdc548.png)

### Steps to reproduce

1. Download .ico file from: https://iconarchive.com/download/i103430/paomedia/small-n-flat/house.ico
2. Save file and mount within app filesystem (I use /app/public/images/favicon.ico
3. Edit settings.yaml to use "favicon: /images/favicon.ico" - save
4. Reload homepage, and still see the default

then

1. Edit settings.yaml to use "favicon: https://iconarchive.com/download/i103430/paomedia/small-n-flat/house.ico" - save
2. Reload homepage, see the .ico in use from url source

### homepage version

v0.5.3

### Installation method

Docker

### Configuration

```yaml
settings.yaml
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

providers:
  openweathermap: openweathermapapikey
  weatherapi: weatherapiapikey

background: /images/background.jpg
backgroundOpacity: 0.4
favicon: https://iconarchive.com/download/i103430/paomedia/small-n-flat/house.ico
```

### Other

Until issue #534 is resolved, none of this matters anyway. Any customization of the settings.yaml leads to i18n key names being used for labels.

**Comment by shamoon** [comment]
I can confirm but honestly I have no idea why its happening. Maybe something to do with how the docker file manages the `/public` directory...

**Comment by TopCheddar27** [comment]
I believe I have found the solution, and it relates to closed issue #534 (as I was also experiencing that). I had my image folder directly mounted into the parent (app/public). After reading the solution to that, I changed my mount point to be /app/public/images and just supplying the files within, and now the favicon.ico seems to be applying correctly. 

Please confirm on your side, but I believe this can be closed.

**Comment by shamoon** [comment]
Ah yea I meant to circle back on this. I could swear I didn’t do that when I setup a test install for this but anyway, I’m sure if it’s solved for you that’s all it was. Thanks
